Directory.Build.targets

```diff
@@ -40,7 +44,7 @@
     <PackageReference Update="Google.Cloud.Firestore" Version="1.1.0" />
 
     <PackageReference Update="Microsoft.Azure.KeyVault" Version="3.0.5" />
-    <PackageReference Update="Microsoft.Data.SqlClient" Version="1.1.3" />
+    <PackageReference Update="Microsoft.Data.SqlClient" Version="2.0.0-preview3.20122.2" />
```

Comment:
I _think_ 2.0.0 is never than 2.0.0-preview3 :P

Reply:
**newer**?

Reply:
newer, yeah, newer.